tests/integration/test_cubepart.py

```diff
@@ -1311,6 +1311,52 @@ def it_uses_columns_proportions_for_ca_cat_x_ca_subvar_population_counts(self):
             np.array([100, 100, 100, 100, 100, 100, 100, 100, 100, 100])
         )
 
+    def it_has_bases_that_dont_sum_across_ca_subvars(self):
+        slice_ = Cube(CR.ca_cat_x_ca_subvar).partitions[0]
```

Comment:
Tipically the fixture name is all CAPITALIZED, please change the name in `CA_CAT_X_CA_SUBVAR` to be compliant with all the tests